models/mat2NV.jl

```
@@ -0,0 +1,30 @@
+using NeuralVerification, MAT
+using NeuralVerification: Layer, Network, ReLU, Id
+
+_vec(A::AbstractMatrix) = vec(A)
+_vec(A::Number) = [A]
+_vec(A::AbstractVector) = A
+
+function readnn(file)
+    vars = matread(file)
+    dic = vars["controller"]
```

Can you throw an error if the key controller does not exist?